src/braille_puzzles.c

```diff
@@ -131,12 +132,12 @@ void UseFlyAncientTomb_Callback(void)
 
 void UseFlyAncientTomb_Finish(void)
 {
-    MapGridSetMetatileIdAt(14, 26, 554);
-    MapGridSetMetatileIdAt(15, 26, 555);
-    MapGridSetMetatileIdAt(16, 26, 556);
-    MapGridSetMetatileIdAt(14, 27, 3634);
-    MapGridSetMetatileIdAt(15, 27, 563);
-    MapGridSetMetatileIdAt(16, 27, 3636);
+    MapGridSetMetatileIdAt(14, 26, METATILE_ID(Cave, SealedChamberEntrance_TopLeft));
+    MapGridSetMetatileIdAt(15, 26, METATILE_ID(Cave, SealedChamberEntrance_TopMid));
+    MapGridSetMetatileIdAt(16, 26, METATILE_ID(Cave, SealedChamberEntrance_TopRight));
+    MapGridSetMetatileIdAt(14, 27, METATILE_ID(Cave, SealedChamberEntrance_BottomLeft) | METATILE_COLLISION_MASK);
```

Comment:
Probably not the best to use the `_MASK` constant here

Reply:
why not?

Reply:
Ironically, it masks what the flags actually represent.

Reply:
so is the solution to replace that constant with the raw value?
and is this a problem everywhere or just here?

Reply:
Create new defines for each collision direction, and a new define that `or`s the 4 directions. Use the new define shifted by `METATILE_COLLISION_MASK`.